db/migrate/20150811023941_create_temp_users.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTempUsers < ActiveRecord::Migration
+  def change
+    create_table :temp_users do |t|
+      t.string :email, default: '', null: false, index: true
```

Comment:
I don't think empty string as default make sense here either. We are validating in the app that they are not empty.